assets/app/index.html

```diff
@@ -112,6 +112,9 @@
     <script src="bower_components/kubernetes-label-selector/labelFilter.js"></script>
     <script src="bower_components/kubernetes-object-describer/dist/object-describer.js"></script>
     <script src="bower_components/openshift-object-describer/dist/object-describer.js"></script>
+    <script src="bower_components/hawtio-utilities/dist/sugar.js"></script>
+    <script src="bower_components/hawtio-utilities/dist/angular-file-upload.js"></script>
```

Comment:
Ah, perhaps not, as hawtio-utilities explicitly adds that module to the plugin loader now that I think about it.